crates/viewer/src/main.rs

```diff
@@ -76,7 +76,7 @@ impl GameApp for ViewerApp {
         let example = Example::parse();
         let shape = example.shape();
 
-        let mesh = shape.mesh();
+        let mesh = shape.mesh().unwrap();
```

Comment:
Let's use an `expect()` here:

```suggestion
        let mesh = shape.mesh().expect("example shape should yield a valid triangulation");
```

Reply:
Good point, fixed.